src/Parser/Core/Modules/Items/BFA/FirstMatesSpyglass.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import { formatPercentage } from 'common/format';
+
+/**
+ * First Mate's Spyglass -
+ * Use: Increase your Critical Strike by 768 for 15 sec. (2 Min Cooldown)
+ */
+class FirstMatesSpyglass extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  
+  casts = 0;
+  timesDamageCrit = 0;
+  timesHealCrit = 0;
+    
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(
+      ITEMS.FIRST_MATES_SPYGLASS.id
+    );
+  }
```

Comment:
For activateable items please also add an entry to the abilities module so it shows in the timeline, cast efficiency and as a possible suggestion. Here's an example of where this is done: https://github.com/WoWAnalyzer/WoWAnalyzer/blob/c9f720358b547ebf2d407efc421ed6bdf3176a29/src/Parser/Core/Modules/Items/Legion/TombOfSargeras/DeceiversGrandDesign.js#L43-L54